src/main/resources/db/migration/V18_02__add_column_fodselsdato.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE person_oversikt_status
+    ADD COLUMN fodselsdato DATE;
```

Comment:
Kan godt legge på en indeks med det samme (siden vi skal bruke denne til å søke etter rader).

Reply:
Prøvde meg på å legge til index på `fodselsdato` kolonnen. Vet ikke om man bør ha én felles index for kolonner man bruker i søk? 🤔

Reply:
Dette er en god start, så kan vi vurdere om flere optimaliseringer trengs senere.